File: src/main/java/com/refinedmods/refinedstorage/util/BlockUtils.java
package com.refinedmods.refinedstorage.util;

import net.minecraft.network.chat.Component;

import net.minecraft.world.level.block.Block;
import net.minecraft.world.level.block.SoundType;
import net.minecraft.world.level.block.state.BlockBehaviour;
import net.minecraft.world.level.material.Material;

public final class BlockUtils {
    public static final BlockBehaviour.Properties DEFAULT_ROCK_PROPERTIES = BlockBehaviour.Properties.of(Material.STONE).strength(0.5F, 6.0F).sound(SoundType.STONE);
    public static final BlockBehaviour.Properties DEFAULT_GLASS_PROPERTIES = BlockBehaviour.Properties.of(Material.GLASS).sound(SoundType.GLASS).strength(0.35F);

    private BlockUtils() {
    }

    // Block#getTranslatedName is client only
    public static Component getBlockTranslation(Block block) {
        return Component.translatable(block.getDescriptionId());
    }
}

File: src/main/kotlin/com/refinedmods/refinedstorage/util/BlockUtils.kt
package com.refinedmods.refinedstorage.util

import net.fabricmc.fabric.api.`object`.builder.v1.block.FabricBlockSettings
import net.minecraft.block.AbstractBlock
import net.minecraft.block.Material
import net.minecraft.sound.BlockSoundGroup

object BlockUtils {
    val DEFAULT_ROCK_PROPERTIES: AbstractBlock.Settings = FabricBlockSettings
            .of(Material.STONE)
            .hardness(1.9f)
            .sounds(BlockSoundGroup.STONE)
    val DEFAULT_GLASS_PROPERTIES: AbstractBlock.Settings = FabricBlockSettings
            .of(Material.GLASS)
            .hardness(0.35f)
            .sounds(BlockSoundGroup.GLASS)
}